src/dbnode/storage/namespace/options_test.go

```diff
@@ -51,6 +53,23 @@ func TestOptionsEqualsIndexOpts(t *testing.T) {
 	require.False(t, o2.Equal(o1))
 }
 
+func getTestSchemaOptions() []*nsproto.SchemaOption {
+	tm := &testproto.TestMessage{}
+	def, _ := tm.Descriptor()
```

Comment:
nit: check error here

Reply:
It does not return error, it always return []{0}, not sure what it is
it is generated protobuf go binding:

` func (*TestMessage) Descriptor() ([]byte, []int) { return fileDescriptorTest1, []int{0} }`

figure it should be ok not to worry about this.